src/tasks/FilterFeedTask.ts

```diff
@@ -74,8 +74,16 @@ export default class FilterFeedTask extends Task {
 
 					if ( this.channel instanceof TextChannel ) {
 						const filterFeedMessage = await this.channel.send( message, embed );
-						NewsUtil.publishMessage( filterFeedMessage );
-						filterFeedMessage.react( this.filterFeedEmoji );
+
```

Comment:
Should `L77` be put in a `try-catch` block?

Reply:
Doesn't need to be, `NewsUtil.publishMessage` doesn't throw